docs/wiki/guides/current/SoftSerial.md

```diff
@@ -0,0 +1,265 @@
+`RESOURCE SOFTSERIAL_TX 1 <pin>`
+
+For Betaflight 4.4 and earlier the equivalent command would be:
+`RESOURCE SERIALTX 11 <pin>`
+
+:::
+
+Softserial requires custom CLI commands. An unused pin with an associated timer must be found, where the original purpose no longer is needed. The most common are PPM or LED_STRIP pins. Not all such pins will work on all boards, and you may need to do some reseach about using softserial on your board before you can make it work.
+
+First go to the CLI and type `resource`, to get a list of the currently assigned pin numbers for each of the currently available pads on the device.
+
+If, for instance, a `LED_STRIP 1` pin is assigned to `A15`, and there is a LedStrip pad on the board, and you are not using it, you could de-assign that pin from `LED_STRIP 1` with the CLI command:
+
+`resource led_strip 1 none`
+
+Then the pin can be assigned to the Softserial Port 1 Tx pin. The commands have changed:
+
+- in Betaflight 4.5 and higher: `RESOURCE SOFTSERIAL_TX 1 A15`
+- in Betaflight 4.4 and lower: `RESOURCE SERIALTX 11 A15`
+
+If the resource reassignment is successful, a `diff` command in CLI for 4.5 will show:
+
+```
+resource LED_STRIP 1 NONE
+resource SOFTSERIAL_TX 1 A15
+```
+
+Note:
+
+- `serialTx` and `serialRx` settings of 11 and 12 are used to configure pins to Softserial ports 1 and 2 respectively.
+- a `diff` command in CLI will show changes from the default resource configuration
+- `resource show all` in CLI will provide a full list of all serial ports, timers etc.
+- just because the resources are re-assigned, it does not mean that the Softserial port will work properly.
+
+Oscarliang provides a good summary of [how to set up softserial](https://oscarliang.com/betaflight-soft-serial/).
+
+### The CLI Serial command.
+
+When Softserial is active, the `serial` CLI command will show the configured serial ports, and a line starting with 30, for Softserial port 1, or 31, for Softserial port 2, will appear.
+
+For example, if Softserial is configured for smartAudio on port 1, a line like this should appear when the `serial` command is sent:
+
+```
+serial 30 2048 115200 57600 0 115200
+```
+
+For more information, see the 'Serial' page in the Development section.
+
+### Historical notes:
+
+The following notes are quite old, and may not be relevant to current versions of Betaflight. They illustrate commands for versions 4.4 and earlier.
+
+CLI commands to assign softserial ports 1 and 2:
+
+`resource serial_tx 11 <pin>`
+`resource serial_rx 11 <pin>`
+
+`resource serial_tx 12 <pin>`
+`resource serial_rx 12 <pin>`
+
+Example; if motors 5 and 6 are not used:
+
+`resource MOTOR 5 NONE`
+
+`resource MOTOR 6 NONE`
+
+:::note
+
+- When assigning softserial pins, the OK means it is compatible, and does not warrant it will work when it is configured as a part of a complete system.
+
+:::
+
+### Software Serial List of compatible pins
+
+OMNIBUS(F3) (by @jflyper)
```

Comment:
F3 is no longer relevant to Betaflight. Suggest this info should go to Archive.

Reply:
You reckon it's best to move the whole original document to archive and start afresh?

Reply:
done!